kola/tests/coretest/core.go

```diff
@@ -98,6 +112,17 @@ func init() {
 			"NTPDate":      TestNTPDate,
 		},
 	})
+	register.Register(&register.Test{
+		Name:             "rhcos.internet",
+		Run:              InternetTests,
+		ClusterSize:      1,
+		ExcludePlatforms: []string{"qemu"},
+		NativeFuncs: map[string]func() error{
+			"CrioPing": TestCrioPing,
+			"CrioEcho": TestCrioEcho,
```

Comment:
Since we are still sorting out how to properly test `crio`, do you want to just make these `TestPodmanPing` and `TestPodmanEcho`?

We could add `crio` specific tests later.

Reply:
It wouldn't hurt anything, through I don't think there is a use case for using `podman` in RHCOS (other than us building for testing).